test/Project.toml

```diff
@@ -1,4 +0,0 @@
-[deps]
```

Comment:
I'm curious about this, I know that both is possible what's the advantage of having the `[extras], [targets]` section instead of `test/Project.toml`?

Reply:
The `test/Project.toml` way is a bit buggy in some situations, unfortunately.